docs/quantum_painter.md

```diff
@@ -730,9 +741,7 @@ The `qp_drawimage` and `qp_drawimage_recolor` functions draw the supplied image
 static painter_image_handle_t my_image;
 void keyboard_post_init_kb(void) {
     my_image = qp_load_image_mem(gfx_my_image);
-    if (my_image != NULL) {
-        qp_drawimage(display, (239 - my_image->width), (319 - my_image->height), my_image);
-    }
+    qp_drawimage(display, (239 - my_image->width), (319 - my_image->height), my_image);
```

Comment:
Removing the NULL check here does not seem to be correct — even if `qp_drawimage()` has its own NULL check inside, there is also the `my_image->height` access here.

The same issue applies to other NULL check removals.